README.rdoc

```diff
@@ -66,6 +66,12 @@ The plugin supports the following parameters:
                             forwarded. If not set, incomplete exceptions stacks
                             are not flushed.
 
+[add_line_break]  Force the addition of a line break '\n' between
```

Comment:
s/add_line_break/force_line_breaks? As mentioned below, we do not always want to add a `\n`.